docker-compose.yaml

```diff
@@ -0,0 +1,73 @@
+version: "3.2"
+
+services:
+  starknet:
+    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-testnet-sharingan-beta.7.1.experimental.4@sha256:baff5a0aa1075d0f84b83ed366f56928b4a6a0c3316762df14e8b1656f784729
+    ports:
+      - "9615:9615"
+      - "9944:9944"
+      - "30333:30333"
+    command:
+      - "--rpc-external"
+      - "--rpc-methods=unsafe"
+      - "--dev"
+    networks:
+      - internal
+
+  kakarot-deployer:
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
+    environment:
+      - ACCOUNT_ADDRESS=0x3 
+      - PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d 
+      # Custom RPC URL for docker
+      - RPC_URL=http://starknet:9944
+      - EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 
+    volumes:
+      # Since STARKNET_NETWORK is not set, deployments will be saved without network folder
+      - deployments:/app/kakarot/deployments
+    depends_on:
+      - starknet
+    restart: on-failure
+    networks:
+      - internal
+
+  deployments-parser:
+    image: jiapantw/jq-alpine:latest@sha256:cacf6adf3463ecfa1315c7bcdf8362fc36a651b071c1e913f048018a380b47cf
+    platform: linux/amd64
+    # Parse deployments and save the result to .env file
+    entrypoint:
```

Comment:
Can we do with this instead?

```
entrypoint: /bin/sh -c
command: |
    echo ...
    echo ...
```

it splits better the image part and what we do with it